### PR TITLE
[expat] Update to 2.6.3

### PR DESCRIPTION
**Pull request by aristotelos**
Version 2.6.3 fixes three critical CVEs.
See https://github.com/libexpat/libexpat/blob/R_2_6_3/expat/Changes

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/expat/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "expat",
-  "version": "2.6.2",
+  "version": "2.6.3",
   "port-version": 1,
```
Comment (JonLiu1993) [inline ports/expat/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/expat/vcpkg.json:4]: When we update port version, please remove "port-version".

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: ports/expat/vcpkg.json (hunk 1 above)
Comment (aristotelos) [reply ports/expat/vcpkg.json]: Good catch... pushed an update

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**